virttest/qemu_monitor.py

```diff
@@ -2373,3 +2373,51 @@ def transaction(self, job_list):
         """
         transaction_args = {"actions": job_list}
         return self.cmd("transaction", transaction_args)
+
+    def dirty_bitmap(self, operation, node, name, granularity=65536):
+        """
+        Add, remove or clear a dirty bitmap.
+
+        :param operation: operations to bitmap, can be 'add', 'remove', and 'clear'
+        :param node: device/node on which to operate dirty bitmap
+        :param name: name of the dirty bitmap to operate
+        :param granularity: granularity to track writes with
+        """
+        cmd = "block-dirty-bitmap-%s" % operation
+        self.verify_supported_cmd(cmd)
+        args = {"node": node, "name": name}
+        if operation.lower() == "add":
```

Comment:
if  operation not in [ add, remove, create] exception will raised in line verify_support_cmd, so covert case here is useless.